src/features/animations/ui/collection-item.tsx

```
@@ -0,0 +1,143 @@
+      {...listeners}
+      aria-label={`Drag ${animation.title} to another collection`}
+      data-pending-move={isPendingMove || undefined}
+    >
+      <Suspense fallback={<Skeleton />}>
+        <button
+          className="flex items-center justify-between text-left gap-2 w-full p-1"
+          onClick={() => {
+            if (dragging) return;
+            onItemSelect(animation);
+          }}
+        >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)